Sources/StreamChatUI/CommonViews/BaseViews.swift

```diff
@@ -344,6 +344,11 @@ open class _ViewController: UIViewController, Customizable {
         updateContent()
     }
     
+    /// `setUp()` is an important function within the ViewController lifecycle
+    /// It's resposibility is to set the delegates and also call `synchronize()`
+    /// this will make sure your local & remote data is in sync.
+    /// If you override this method without calling `super.setUp()` it's important
+    /// to make sure `synchronize()` is called.
```

Comment:
@adamrushy to be more explicit about the intent here, this paragraph could probably leverage the [important markup callout](https://developer.apple.com/library/archive/documentation/Xcode/Reference/xcode_markup_formatting_ref/Important.html#//apple_ref/doc/uid/TP40016497-CH37-SW1).

Reply:
Nice idea 👍🏼